docs/quick_tutorial/authentication.rst

```diff
@@ -101,6 +101,31 @@ Steps
 
 #. Click the "Logout" link.
 
+#. Lets test if only the logged one can see ``Logout`` message:
+
+   .. literalinclude:: authentication/tutorial/tests.py
+    :linenos:
+
+#. Add  ``pyramid[pyramid]`` as a test dependency in ``authentication/setup.py``:
+
+   .. literalinclude:: authentication/setup.py
+    :language: python
+    :emphasize-lines: 11
+    :linenos:
+
+#. Install testing dependency:
+
+   .. code-block:: bash
+
+       $VENV/bin/pip install -e ".[testing]"
+
+#. Run your Pyramid application tests with:
+
+   .. code-block:: bash
+
+       $VENV/bin/pytest tutorial/tests.py
+
+
```

Comment:
Please rebase your changes on (or manually compare to) `master`, per the updates I recently made throughout the Quick Tutorial's setup.py's and narrative documentation, then push your changes to this branch.